website/versioned_docs/version-0.60/native-modules-intro.md

```diff
@@ -0,0 +1,26 @@
+---
+id: version-0.5-native-modules-intro
```

Comment:
`id` parameter is incorrect, also `original_id` parameter is redundant.

Also if you would like to update the versioned docs for `0.60` you also need to update the docs for all versions in between (`0.61`, `0.62`).

Reply:
@Simek removing this, I am only adding native-modules-intro + the new docs it links to in 0.63